crates/hir-expand/src/fixup.rs

```diff
@@ -601,20 +601,6 @@ fn foo () {a . b ; bar () ;}
         )
     }
 
-    #[test]
-    fn extraneous_comma() {
-        check(
-            r#"
-fn foo() {
-    bar(,);
-}
-"#,
-            expect![[r#"
-fn foo () {__ra_fixup ;}
-"#]],
-        )
-    }
```

Comment:
That `bar(,)` has a parse error is the whole point of the test :smile: previously the syntax fixup removed the whole call, probably it now doesn't do that anymore because there's no `ERROR` node. Ideally we'd probably do something like turn `bar(,)` into` bar(__ra_fixup,)`. We need to do _something_ since proc macros will not accept `bar(,)`.

Reply:
Thanks!
Um, I suppose I need to look into the syntax fixup a bit more.